cgi-bin/LJ/S2/ReplyPage.pm

```diff
@@ -57,7 +57,9 @@ sub ReplyPage {
     # canonical link to the entry or comment thread
     $p->{head_content} .= LJ::canonical_link( $entry->url, $replytoid );
 
-    $p->{'head_content'} .= $LJ::COMMON_CODE{'chalresp_js'};   # Hey, I think this is destroyed? -NF
```

Comment:
Lol thank you for dropping the other shoe. 

But I'm like 98% sure MD5 can be removed entirely from ReplyPage; there's no case in which it will get used. @momijizukamori, want to chime in?

Reply:
Thanks for taking a look! My understanding of JS is shaky at best so I didn't want to make any assumptions, but I had a sense that might be the case. Of course right now it's doing exactly what it was before, only more obviously.